roxctl/image/scan/scan.go

```diff
@@ -207,6 +216,17 @@ func (i *imageScanCommand) Validate() error {
 				"only specify json or csv", i.format)
 		}
 	}
+
+	validSeverities := []string{lowCVESeverity.String(),
```

Comment:
This makes the severity case sensitive, right? I think we should allow both `--severity important,critical` and `--severity IMPORTANT,CRITICAL`. So I'd suggest to do some input normalization.